app/helpers/application_helper.rb

```diff
@@ -33,9 +33,7 @@ def loading_tag(**args)
   end
 
   def quick_switch_tag(image_url, short_text, hover_name, char_id)
-    if image_url.nil?
-      return content_tag :div, short_text, class: CHAR_ICON_FAKE, title: hover_name, data: { character_id: char_id }
-    end
+    return content_tag :div, short_text, class: CHAR_ICON_FAKE, title: hover_name, data: { character_id: char_id } if image_url.nil?
```

Comment:
Surprised this doesn't trigger a line width complaint but I guess not 🤷 my usual check is if it's readable on Github, which this isn't but honestly not a big deal.

Reply:
It's actually like 15 characters short of the max, apparently